packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -214,19 +214,27 @@ export const Input = withLocalization(UnwrappedInput);
 
 type StripeElementProps = { onValidate?: OnValidateFunction } & FieldProps & {
     component: any;
+    getString: Function | undefined;
   } & ReactStripeElements.ElementProps;
 
 export const defaultStripeElementValidator: OnValidateFunction = (
   value,
   focused,
-  props
+  props,
+  getString
 ) => {
   if (!value || value.empty) {
     if (props.required) {
       return {
         value,
         valid: false,
-        error: focused ? null : `${props.label} is required`,
+        error: focused
+          ? null
+          : `${
```

Comment:
Don't really need the backtick-string at this point - but 🤷‍♂️

Reply:
removed the backticks, glad you caught that. I'll merge once we are green again